observability/logging/cluster-logging-deploying.adoc

```diff
@@ -7,57 +7,53 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
+
+[TIP]
+====
+You can alternatively apply all example objects.
 ====
 
 ifdef::openshift-origin[]
 [id="prerequisites_cluster-logging-deploying"]
 == Prerequisites
 * Ensure that you have downloaded the {cluster-manager-url-pull} as shown in _Obtaining the installation program_ in the installation documentation for your platform.
 +
-If you have the pull secret, add the `redhat-operators` catalog to the OperatorHub custom resource (CR) as shown in _Configuring {product-title} to use Red Hat Operators_.
+If you have the pull secret, add the `redhat-operators` catalog to the OperatorHub custom resource (CR) as shown in _Configuring {product-title} to use Red{nbsp}Hat Operators_.
 endif::[]
 
-//Installing the CLO via webconsole
-include::modules/cluster-logging-deploy-console.adoc[leveloffset=+1]
-include::modules/create-cluster-logging-cr-console.adoc[leveloffset=+1]
+include::modules/logging-es-deploy-console.adoc[leveloffset=+1]
 
-// Install using CLI
-include::modules/cluster-logging-deploy-cli.adoc[leveloffset=+1]
-include::modules/create-cluster-logging-cli.adoc[leveloffset=+1]
+include::modules/logging-es-deploy-cli.adoc[leveloffset=+1]
 
-[id="cluster-logging-deploying-postinstallation"]
-== Postinstallation tasks
+--
+include::snippets/logging-retention-period-snip.adoc[leveloffset=+1]
+--
 
-After you have installed the {clo}, you can configure your deployment by creating and modifying a `ClusterLogging` custom resource (CR).
+include::modules/logging-loki-cli-install.adoc[leveloffset=+1]
 
-[TIP]
-====
-If you are not using the Elasticsearch log store, you can remove the internal Elasticsearch `logStore` and Kibana `visualization` components from the `ClusterLogging` custom resource (CR). Removing these components is optional but saves resources. See xref:../../observability/logging/log_storage/logging-config-es-store.adoc#cluster-logging-removing-unused-components-if-no-elasticsearch_logging-config-es-store[Removing unused components if you do not use the Elasticsearch log store].
-====
-
-include::modules/cluster-logging-about-crd.adoc[leveloffset=+2]
-include::modules/configuring-log-storage-cr.adoc[leveloffset=+2]
-include::modules/configuring-logging-collector.adoc[leveloffset=+2]
-include::modules/configuring-log-visualizer.adoc[leveloffset=+2]
-include::modules/cluster-logging-deploy-multitenant.adoc[leveloffset=+2]
+include::modules/logging-loki-gui-install.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
 ifdef::openshift-enterprise,openshift-origin[]
-* xref:../../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy]
+* xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[About the OpenShift SDN default CNI network provider]
+* xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[About the OVN-Kubernetes default Container Network Interface (CNI) network provider]
```

Comment:
These new links appear to be repeats of existing links. 

![image](https://github.com/user-attachments/assets/9fe7ae51-e5be-445e-9cce-770442ac901f)